block_function.rb

```diff
@@ -0,0 +1,6 @@
+def do_calc
+    yield(7, 9)
+  end
+  
+  puts do_calc { |a, b| a + b }
+  puts do_calc { |a, b| a * b }
```

Comment:
The assignment reads:

> do_calc will print out the result returned by the yield

which I would interpret as this:

```suggestion
def do_calc
    puts yield(7, 9)
  end
  
  do_calc { |a, b| a + b }
  do_calc { |a, b| a * b }
```

But it's not a particularly important part of this assignment.